packages/strapi-plugin-content-type-builder/admin/src/translations/en.json

```diff
@@ -171,6 +171,9 @@
   "popUpWarning.bodyMessage.category.delete": "Are you sure you want to delete this category? All the components will also be deleted.",
   "popUpWarning.bodyMessage.component.delete": "Are you sure you want to delete this component?",
   "popUpWarning.bodyMessage.contentType.delete": "Are you sure you want to delete this collection type?",
+  "popUpWarning.draft-publish.button.confirm": "Yes, disable",
```

Comment:
Is it `Yes, disable` or `Yes, continue`?
@JabStrapi

Reply:
I think "Yes, disable" is the right word since the modal is triggered when trying to disable a toggle input.